platformio.ini

```diff
@@ -27,7 +28,7 @@ lib_deps =
 	https://github.com/OXRS-IO/OXRS-IO-API-ESP32-LIB
 	https://github.com/OXRS-IO/OXRS-IO-IOHandler-ESP32-LIB
 build_flags = 
-	-DFW_NAME="${firmware.name}"
+  ;	-DFW_NAME   set by debug/release_extra.py
```

Comment:
Not a fan of commenting out (and leaving) code - better to remove and leave the original in the git logs.

Reply:
This wasn't meant to comment out unused code, but to be a hint for others where to find the kind of hidden definition of FW_NAME.
We can remove it if you like that better.

Reply:
Yeah understood, happy with it as is then.